cypress/integration/unversioned.spec.ts

```diff
@@ -0,0 +1,21 @@
+const resizeObserverLoopErrRe = /> ResizeObserver loop/;
+Cypress.on("uncaught:exception", (err) => {
+  if (resizeObserverLoopErrRe.test(err.message)) {
+    return false;
+  }
+});
```

Comment:
Maybe `(err) => !err.message.includes("ResizeObserver loop")` would be simpler? Or does it catch more cases right now (not sure about the ">" sign at the beginning)?

Reply:
Yep.. Not sure why I used a regexp in the first test, I removed it from there, I do not think it should be fired, and modified the original code. 🤞

Reply:
Cool!